tests/UnitTests/CoreGraphics.drawing/CGContextDrawingTests.cpp

```diff
@@ -272,3 +272,37 @@ DRAW_TEST_F(CGContext, ClipBoundingBox, WhiteBackgroundTest<>) {
     CGContextSetRGBFillColor(context, 0.0, 1.0, 0.0, 1.0);
     CGContextFillRect(context, CGContextGetClipBoundingBox(context));
 }
+
+DRAW_TEST_F(CGContext, SubPixelAlignment, WhiteBackgroundTest<>) {
```

Comment:
How do you mean? This uses a transform to align the rectangle to the half pixel mark. Are you just looking for redundancy with other types of transforms?